docs/Key-Manager-Support.md

```diff
@@ -44,6 +90,28 @@ Once configured, the Key Manager will generate and manage keys tied to the uploa
  - Enables the Key Manager to use it for signing VCs.
  - Integrates the certificate into the ongoing key lifecycle.
 
+## Sequence Diagram for CSR generation and Certificate Upload 
+```mermaid
+sequenceDiagram
+    participant Client
+    participant Certify System
+    participant KeyManager Service
```

Comment:
the wording certify system and keymanager service, sounds like keymanager is a different container.
We can rename to Inji Certify API and Keymanager library